src/lib/fees/model.ts

```diff
@@ -11,7 +16,8 @@ export const THORSWAP_MAXIMUM_YEAR_TRESHOLD = 2025
 type CalculateFeeBpsArgs = {
   tradeAmountUsd: BigNumber
   foxHeld: BigNumber
-  thorHeld?: BigNumber
+  thorHeld: BigNumber
+  foxWifHatHeld: BigNumber
```

Comment:
```suggestion
  foxWifHatHeldCryptoBaseUnit: BigNumber
```

Reply:
Done in [83851c8](https://github.com/shapeshift/web/pull/8663/commits/83851c8d6f6af8485ce7e3c2c29be7f518b42cc1)